14/umbraco-cms/extending/packages/language-files-for-packages.md

```diff
@@ -2,39 +2,59 @@
 description: "Information on how to use language files to make your Umbraco package UI support multiple languages"
 ---
 
-# Language file for packages
-
-Umbraco Core includes language files, but package authors must provide their own for multi-lingual UI.
-
-## Including new language keys
-
-For each language your package supports, you include an .xml file in the same format as the core language files, named with its language code. The language files must be located in a `Lang` folder inside your package folder in `App_Plugins`. If your package assets are in `/App_Plugins/mypackage` all language files must be placed in the following locations:
-
-- English keys: `/App_Plugins/mypackage/Lang/en-US.xml`
-- Danish keys: `/App_Plugins/mypackage/Lang/da-DK.xml`
-
-{% hint style="info" %}
-The `App_Plugins` version of the `Lang` directory is case sensitive on Linux systems, so make sure that it start with a capital `L`.
-{% endhint %}
+# Languages for packages
+
+If you want your package to be available in different languages, you can use the existing localizations from Umbraco or register your own localizations. The localizations are written as a key-value pair pattern.
```

Comment:
This section all looks correct to me, but seems to repeat what is already detailed here:
https://docs.umbraco.com/umbraco-cms/extending/language-files/ui-localization

So in the interests of maintenance, it might be better to just have the first sentence here and then link off to this content rather than repeating it.

Reply:
@AndyButland Good point!

I think it still makes sense to have the first example on this page, however I have removed all the other and instead links to the UI localization page instead :-)